### ISSUE TITLE
Always has "0 passes".

### ISSUE THREAD

**Issue by wieczyk** [privacypass/challenge-bypass-extension#141]
I always have 0 passes. Clicking "Get More Passes" takes me to captcha website and then
* I cannot get pass through captcha. Many tries gives nothing.
* Sometimes I am verified as human and then I go to privacypass.github.io. Although verification was successful I still have 0 passes.

I am asked about verification almost on every page now. Even if I verify my self I am asked again in two minutes.

**Comment by fflide** [comment]
Hi. I have this problem too. It started about 15 hours ago. Two computers (windows 7, limux), browsers: firefox-esr 60.8.0, firefox 70.0.1, chrome 78.0.3904.108. everywhere one result - "passes 0".

**Comment by wieczyk** [comment]
Check your IP on https://www.projecthoneypot.org/search_ip.php

I am behind NAT and it looks like someone from my network has infected a computer.

**Comment by alxdavids** [comment]
Hi, thanks for bringing this up, it does indeed seem like a problem that was introduced recently and I will investigate it today.

**Comment by alxdavids** [comment]
It seems that Cloudflare has changed the URL used to send CAPTCHA solutions. I've written a change that should fix this. Once this is merged I'll release a new version of the extension with the fix.

**Comment by fflide** [comment]
![изображение](https://user-images.githubusercontent.com/58248628/69739556-f4c39f80-1148-11ea-88cc-fd1595b11561.png)

**Comment by fflide** [comment]
When a patched version appears on firefox?

**Comment by alxdavids** [comment]
Hopefully in the next couple of days, Chrome may take a little longer

**Comment by fflide** [comment]
Thank you. I build and sign .xpi, all works fine )

**Comment by alxdavids** [comment]
This shouldn't have been closed, the fix has been merged but yet to release to the extension store.

**Comment by 4evermaat** [comment]
So is there a non-chrome store extension file that I can get from here, manually add to the browser, and that I can test?

(same for firefox also)

This would help.

And sorry for creating duplicate ticket.   I did do a search first.

**Comment by alxdavids** [comment]
Version 2.0.4 has been released to the FF store and that should include a fix for this issue. You should be able to test that by updating your extension from the store. Releasing a fix to the Chrome store may take a little while longer.

**Comment by alxdavids** [comment]
@4evermaat You can also try building from the master branch and installing directly into the browser if you want to try something in Chrome.

**Comment by 4evermaat** [comment]
> @4evermaat You can also try building from the master branch and installing directly into the browser if you want to try something in Chrome.

@alxdavids how would I build for Chrome?   Is there some instructions to do this?

<p/>
<br />
<br />

---



> Version 2.0.4 has been released to the FF store and that should include a fix for this issue. You should be able to test that by updating your extension from the store.

I see it was updated 18 minutes ago in FF addons database, lol.   I deleted and reinstalled.   And voila:

![Snag_506a7e55](https://user-images.githubusercontent.com/830306/69823661-21a4af00-123c-11ea-9779-15192c216066.png)



I tried it twice, and got 30 passes each time.

**Comment by alxdavids** [comment]
@4evermaat You can try these [instructions](https://github.com/privacypass/challenge-bypass-extension#development) from the README. Let me know if they're not clear enough.

**Comment by 4evermaat** [comment]
> @4evermaat You can try these [instructions](https://github.com/privacypass/challenge-bypass-extension#development) from the README. Let me know if they're not clear enough.

what are you using to execute [`quickstart`](https://github.com/privacypass/challenge-bypass-extension#quickstart)??

Maybe a link and/or screenshot would help.    I'm not a coder, although I do manage some coding projects.

**Comment by 4evermaat** [comment]
Or what would be better also would be to just zip/attach a built one here that I can add manually into chrome.  

You can delete it when the chrome store is updated.

**Comment by 4evermaat** [comment]
> @alxdavids
> Or what would be better also would be to just zip/attach a built one here that I can add manually into chrome.
> 
> You can delete it when the chrome store is updated.

Or anybody really???  The chrome store updates are very slow to update sometimes.

**Comment by alxdavids** [comment]
Version 2.0.4 is now available in the Chrome store, so you should be abl eto download it directly from there (if it hasn't updated already).

Going to close this now as the problem should be fixed.

**Comment by lestephane** [comment]
Looks like it's working now in Firefox for me. Thanks

**Comment by OlegBrony** [comment]
21/09/20. still not working. thanks -_-

**Comment by alxdavids** [comment]
@OlegBrony It appears to work fine for me. If it does not work in your case, it is probably not related to this previous issue which was fixed. Feel free to create a new issue with details about your setup and the problem, then we may be able to look into it.
